homeassistant/components/devolo_home_control/strings.json

```diff
@@ -45,5 +45,16 @@
         "name": "Brightness"
       }
     }
+  },
+  "exceptions": {
+    "connection_failed": {
+      "message": "Failed to connect to devolo Home Control central unit {gateway_id}."
+    },
+    "invalid_auth": {
+      "message": "Authentication failed. Please re-authenticaticate with your mydevolo account."
+    },
+    "maintenance": {
+      "message": "devolo Home Control is currently in maintenance mode."
```

Comment:
is devolo with lowercase d intentional?

Reply:
Yes, that is the official company spelling.